src/main/java/com/example/java_pro_homework_7/Pet/Dog.java

```diff
@@ -0,0 +1,43 @@
+package com.example.java_pro_homework_7.Pet;
+
+public class Dog extends Animal {
+
+    public static int dogCounter;
+
+    public Dog(String name, String color, String breed) {
+        super(name, color, breed);
+        limitDistanceSwim = 10;
+        limitDistanceRun = 500;
+        getDogCounter();
+        dogCounter++;
+    }
+
+    static public int getDogCounter() {
```

Comment:
Would be better to declare modifiers as following -> `public static` instead of `static public`